rails-commenting.rb

```diff
@@ -7,24 +7,31 @@
 # FILE: app/controller/blog_posts_controller.rb
 
 # ---1)
+#The first step is to create a controller method that will access all the items from the database.
 class BlogPostsController < ApplicationController
   def index
     # ---2)
+    #In the index method we are making an Active Record call that will save all the content from the database as an instance variable.
     @posts = BlogPost.all
   end
 
   def show
     # ---3)
+    #Show is a RESTful route that looks for one item in the database.
```

Comment:
true but the line underneath is creating a variable that specifically fins one entry in the Database according to the number passed by params